### ISSUE TITLE
How to register a custom field with multiple selections

### ISSUE THREAD

**Issue by onozaty** [onozaty/redmine-issue-loader#10]
* https://blog.enjoyxstudy.com/entry/2020/05/02/000000#comment-13574176438093811602
```
１，複数選択可としたカスタムフィールド（リスト形式）
　　に対してのCREATE・UPDATE方法につきまして
　　ご教授いただけますでしょうか?
```

**Comment by onozaty** [comment]
Multiple values are not supported at this time.
I would like to investigate how to respond to this issue.

**Comment by onozaty** [comment]
Released at v2.4.0
